### PR TITLE
Add page describing SLAs and support process

### PR DESCRIPTION
**Pull request by cghsystems**

### CODE REVIEW

File: docs/ske/04-support/00-intro.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+## Support Channels
+Technical support is available exclusively through the Syntasso support channels listed in your license agreement
+
+### Service Level Agreements 
+
+| Service Level | Description | Response |
+| ----- | ------- |----------- |
+| **Level 1** | Release is unusable or substantially inoperable as a result of an Incident | Response within 4 Hours, Fix within 5 business days |
+| **Level 2** | Release has an Incident which has an adverse impact on its use or functionality | Response within 1 business day, Fix within 10 business days |
+| **Level 3** | Release has a minor problem which can be worked round by the member of the Bank Group concerned without significant inconvenience. | Fix timeline published on roadmap |
```
[the remark this answers is not in the document]
Comment (abangser) [reply docs/ske/04-support/00-intro.mdx:25]: And should this have an ack time frame?

File: docs/ske/04-support/00-intro.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Unofficial Channels
+
+Syntasso support will respond to all requests received through the official channels listed. However, community channels exist in the Kratix Slack Workspace [TODO ADD LINK] for peer-to-peer support and discussion.
```
Comment (abangser) [inline docs/ske/04-support/00-intro.mdx:29]: missing link!

**Review by abangser — CHANGES_REQUESTED** [review]
Made a few updates/comments. Also curious about having a index page for this given there is only one piece of content. I would suggest we link straight to it until we have a second page. WDYT?

File: docs/ske/04-support/00-intro.mdx (hunk 2 above)
Comment (cghsystems) [reply docs/ske/04-support/00-intro.mdx]: Goo spot. Added.

File: docs/ske/00-support.mdx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+SKE Support is available during Syntasso's business day between the hours of 9:30-17:30 UK time. If you require support over and above these times please contact us for more information on our Premium Support package. 
+
+## Support Channels
+Technical support is available exclusively through the Syntasso support channels listed in your license agreement
+
+### Service Level Agreements 
+
+| Service Level | Description | Response |
+| ----- | ------- |----------- |
+| **Level 1** | Release is unusable or substantially inoperable as a result of an incident | Response within 4 business Hours, Fix within 5 business days |
```
Comment (abangser) [inline docs/ske/00-support.mdx:22]:
```suggestion
| **Level 1** | Release is unusable or substantially inoperable as a result of an incident | Response within 4 business hours, Fix within 5 business days |
```

**Review by abangser — APPROVED** [review]
just one errant capitalisation. otherwise renders well and looks great 💅

**Pull request merged**